### ISSUE TITLE
Redirect the files and modules routes

### ISSUE THREAD

**Issue by rowanmanning** [Financial-Times/origami-build-service#11]
For consistency with bundles, should we add the following redirects?

```
/files/* > /v2/files/*
/v1/files/* > /v2/files/*
/modules/* > /v2/modules/*
/v1/modules/* > /v2/modules/*
```

The code that handles these is exactly the same. Could we see this causing any issues, apart from the additional load of the redirects. Can we get logs for these routes?

**Comment by AlbertoElias** [comment]
We can get logs, but there should be no issues whatsoever, and I'm in favour of redirecting. We can also start working on the whitelist we talked with Andrew about of current sites using an unversioned url, and any others who try to use it, will get a 404. I'll raise a separate issue

**Comment by rowanmanning** [comment]
This has been added in #50

**Comment by rowanmanning** [comment]
Released as part of `3.6.0`
